stacker/tests/lookups/handlers/test_kms.py

```diff
@@ -9,18 +13,20 @@
 
 class TestKMSHandler(unittest.TestCase):
     def setUp(self):
-        self.plain = "my secret"
+        self.plain = b"my secret"
         with mock_kms():
             kms = boto3.client("kms", region_name="us-east-1")
             self.secret = kms.encrypt(
                 KeyId="alias/stacker",
-                Plaintext=self.plain.encode("base64")
+                Plaintext=codecs.encode(self.plain, 'base64').decode('utf-8'),
             )["CiphertextBlob"]
+            if isinstance(self.secret, bytes):
+                self.secret = self.secret.decode()
 
     def test_kms_handler(self):
         with mock_kms():
             decrypted = handler(self.secret)
-            print "DECRYPTED: %s" % decrypted
+            print("DECRYPTED: %s" % decrypted)
```

Comment:
This looks like a leftover debug statement - we can probably remove it, though not necessary here.

Reply:
OK, I'll remove it.